src/components/rangeslider/draw.js

```diff
@@ -92,8 +95,13 @@ module.exports = function(gd) {
         opts._height = (fullLayout.height - margin.b - margin.t) * opts.thickness;
         opts._offsetShift = Math.floor(opts.borderwidth / 2);
 
-        var x = margin.l + (graphSize.w * domain[0]),
-            y = fullLayout.height - opts._height - margin.b;
+        var x = margin.l + (graphSize.w * domain[0]);
+
+        var y = (
+            margin.t + graphSize.h * (1 - oppDomain[0]) +
+            tickHeight +
+            opts._offsetShift + constants.extraPad
+        );
```

Comment:
Also, this isn't new but I'm just noticing it now while we're here: should we switch `handleStroke` to be `Color.defaultLineColor = '#444'` instead of `#666` as it is now? And to avoid duplication, `handleFill = '#fff'` is already available as `Color.background` - to explicitly match how we define [zoombox corners](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/dragbox.js#L202-L203)